app/controllers/users_controller.rb

```diff
@@ -122,27 +123,23 @@ def list
       end
     end
     # allow admins to view recent users
-    @users = if params[:id]
-               User.order(order_string)
-                             .where('rusers.role = ?', params[:id])
-                             .where('rusers.status = 1')
-                             .page(params[:page])
-
-             elsif @tagname_param
-               User.where(id: UserTag.where(value: @tagname_param).collect(&:uid))
-                             .page(params[:page])
-
-             else
-               # recently active
-               User.select('*, rusers.status, MAX(node_revisions.timestamp) AS last_updated')
-                            .joins(:revisions)
-                            .where("node_revisions.status = 1")
-                            .group('rusers.id')
-                            .order(order_string)
-                            .page(params[:page])
-             end
-
+    if params[:id]
+      @users = User.order(order_string)
+        .where('rusers.role = ?', params[:id])
+        .where('rusers.status = 1')
+    elsif @tagname_param
+      @users = User.where(id: UserTag.where(value: @tagname_param).collect(&:uid))
+    else
+      # recently active
+      group_by = Rails.env.development? ? false : 'rusers.id' # for GitPod compatibility; #8117
```

Comment:
here trying to call `group()` with either `false` or `rusers.id`... but it's not working it seems... or, maybe the `false` is working? Need to try running it carefully to diagnose, but i have to go now...